### PR TITLE
show how many people are online in server browser

### PR DESCRIPTION
**Pull request by sornas**

### CODE REVIEW

File: sylt/sylt-std/src/sylt.rs [hunk 1]
```diff
@@ -64,6 +64,30 @@ pub fn map(ctx: RuntimeContext<'_>) -> Result<Value, RuntimeError> {
...
+            return Ok(Value::List(Rc::new(RefCell::new(filtered))));
+        }
+        _ => {}
+    }
+
+    return Err(RuntimeError::ExternTypeMismatch(
+        "map".to_string(),
+        values.iter().map(Type::from).collect(),
+    ));
+}
```
Comment (FredTheDino) [inline sylt/sylt-std/src/sylt.rs]: :heart_eyes:

File: sylt/sylt-std/src/sylt.rs [hunk 2]
```diff
@@ -266,6 +290,15 @@ sylt_macro::extern_function!(
     },
 );
 
+sylt_macro::extern_function!(
+    "sylt_std::sylt"
+    as_str
```
Comment (FredTheDino) [inline sylt/sylt-std/src/sylt.rs:342]: +1

**Review by FredTheDino — APPROVED** [review]

File: game.sy [hunk 3]
```diff
@@ -180,8 +181,18 @@ clear_input :: fn input: entity.Input -> entity.Input {
 }
 
 process_player_input :: fn delta: float {
+    players -> for_each' fn player: entity.Player {
+        entity.player.remove = entity.player.last_seen >= 3.0
+    }
+    players = players -> filter' fn player: entity.Player -> bool {
+        !player.remove
+    }
```
Comment (FredTheDino) [inline game.sy]: Won't this shift all the player IDs? This should be a dict.

File: game.sy [hunk 4]
```diff
@@ -180,8 +181,18 @@ clear_input :: fn input: entity.Input -> entity.Input {
 }
 
 process_player_input :: fn delta: float {
+    players -> for_each' fn player: entity.Player {
+        entity.player.remove = entity.player.last_seen >= 3.0
```
Comment (FredTheDino) [inline game.sy]: Break out the 3.0 to a variable. Maybe disconnect time?

**Review by EMattfolk — APPROVED** [review]

**Pull request merged**